app/javascript/components/pages/EventShow.js

```diff
@@ -6,13 +6,31 @@ class EventShow extends Component {
     this.state = {
       loading: true,
       event: undefined,
+      loadingDrink: true,
+      drink: [],
     }
   }
 //
 componentDidMount(){
-  this.eventShow()
+  this.eventShow();
+  this.getDrink()
 }
 
+  getDrink = () => {
+    fetch('https://www.thecocktaildb.com/api/json/v1/1/random.php')
+    .then((response) => {
+      if(response.status === 200){
+        return response.json();
+      }
+    })
+    .then((payload)=> {
+      this.setState({drink: payload, loadingDrink: false})
```

Comment:
initializing would be ideal to store one drink in state since you'll only need one drink to display